src/components/solvers/SolveRequest.ts

```diff
@@ -0,0 +1,17 @@
+export interface SubSolveRequest {
+    /**
+     * If no solver id is provided, the toolbox choose the solver itself via meta solver strategy
+     */
+    requestedSolverId?: string,
+    /**
+     * Map from problem type to SubSolveRequest to allow explicitly requested solvers for a subroutine
+     */
+    requestedSubSolveRequests: Map<string, SubSolveRequest>
+}
+
+export interface SolveRequest<T> extends SubSolveRequest {
+    requestContent?: T;
+    /**
+     * If no solver id is provided, the toolbox choose the solver itself via meta solver strategy
+     */
```

Comment:
redundant comment

Reply:
Why is it redundant? Imo it helps someone who isn't familiar with this type to see that they don't necessarily have to set a solver id and what happens if they don't set it and set it to the toolbox.

Reply:
The comment is redundant as it describes the `requestedSolverId` attribute which is (along with its comment) inherited from `SubSolveRequest`. `requestedSolverId` already has the exact same comment.

The comment is useful but I don't see any use in writing it down twice.

Reply:
Oh I see. Sorry, I didn't see it in the context.